sta_exercise_final.ipynb

```diff
@@ -0,0 +1,985 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    merged = merge_spiks_and_stims(spiks, stims, dt)
</pre><p>I don't understand the logic here: why would you merge these into the same matrix, when you multiply them together anyway. This works, but general comment is to keep robs (response) separate from the stimulus.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NeuroTheoryUMD/mattjac/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Originally, it was a convenience before I knew I needed to look at lagged stims, and not just look at stims--&gt;responses. I will remove it and use the "design matrix" you mentioned instead.</p>